.storybook/custom/custom.scss

```diff
@@ -491,3 +491,27 @@ body.sb-show-main {
   border: 1px dashed var(--sapTextColor) !important;
   border-radius: 0.25rem;
 }
+
+.fd-grid-list__item--body {
```

Comment:
There shouldn't be any `fd-` classes in this file - just playground classes. That way the styles don't get confusing whether it's part of the actual component or just for the documentation.

Reply:
fixed